src/main.rs

```diff
@@ -58,7 +56,7 @@ fn main() {
     let video_subsystem = sdl_context.video().unwrap();
     let window = video_subsystem
         .window(
-            "Critter's Amazing Chip8 Emulator",
+            "PTC's Working Chip8 Emulator",
```

Comment:
I believe that in the parlance of Twitter, this is called "throwing shade."

Reply:
forgot about that lol, was my first change just to see how rust compiles/runs stuff